services/workflows-service/src/business/business.controller.internal.ts

```diff
@@ -39,6 +48,46 @@ export class BusinessControllerInternal {
     );
   }
 
+  @common.Get('sync')
+  @common.UseGuards(AdminAuthGuard)
+  @ApiExcludeEndpoint()
+  async getAllBusinesses() {
```

Comment:
What this endpoint is used for?

Reply:
To import the businesses fom the Unified API